### DISCUSSION TITLE
In 3.4.1 all dashboards are empty

### DISCUSSION THREAD

**Discussion by Airwave1981** [flobz/psa_car_controller#790] [Q&A] [answered]
Hi all,

after I got a new login done in 3.4.1 now all dashboards are empty.

What issue could that be?

Thanks in advance.

![image](https://github.com/flobz/psa_car_controller/assets/32637309/c6a0c33c-40e2-448b-ba5c-30b5dc5c979c)
![image](https://github.com/flobz/psa_car_controller/assets/32637309/a1844f30-04ed-49a5-beb6-790d12ea7673)
![image](https://github.com/flobz/psa_car_controller/assets/32637309/cc1854f1-cbae-4512-8c93-37576eefa0ed)
![image](https://github.com/flobz/psa_car_controller/assets/32637309/8c4990f6-8863-430c-a595-8e05ee9005a2)

**Comment by bennymamo** [comment]
Is it a new install? I used an old info.db and it looked exactly like that. It took like 30 mins for data to be shown. But all data is back.

**Reply by Airwave1981** [reply]
Yes it is a new install. Than I wait a bit longer...

**Reply by Airwave1981** [reply]
Strange, 4 hours later, one drive also with the car. Nothing changed. Everything empty?!?
Could that be?

**Reply by Airwave1981** [reply]
In the meantime i see often an error 
AttributeError: 'NoneType' object has no attribute 'get_energy'
in the log.

And still all dashboards are empty.

**Comment by Airwave1981** [comment, marked as answer]
Okay, I reinstalled via docker in Linux and the issue is gone. Just occured under windows. 

So seems a windows only  issue.occurred
